data_structures/stack_using_queue.cpp

```diff
@@ -0,0 +1,106 @@
+/**
+ * @file
+ * @brief Implement a last in first out (LIFO) stack using queues.
+          The implemented stack should support all the functions of a normal queue (push, top, pop, and empty).
+ * @author [Neha Labhasetwar](https://github.com/nehalabhasetwar)
+ */
+
+#include <iostream>
+#include <queue>
+#include <cassert>
```

Comment:
Add a brief description of what the header `queue` does (what does it add).
```suggestion
#include <iostream>  /// for io operations
#include <queue>    /// description
#include <cassert>  /// for assert
```